src/by/naumenka/lesson36/Document.java

```diff
@@ -0,0 +1,29 @@
+package by.naumenka.lesson36;
+
+class Document {
+    private String documentNumber;
+    private String creationDate;
+    private String nameAuthor;
+
+
+    public void setDocumentNumber(String documentNumber) {
```

Comment:
Getters and setters MUST present in pair for each private/protected field (even if some of them are not needed)